devtools/conda-envs/fep_env.yaml

```diff
@@ -21,3 +21,6 @@ dependencies:
   - pytest
   - pytest-cov
   - codecov
+    # Pip-only installs
+  - pip:
+     - git+https://github.com/wiederm/transforato_tetsystems.git
```

Comment:
@wiederm Is there a reason for the strange name?

Reply:
thanks, corrected the mistake!